src/App.js

```diff
@@ -14,11 +14,87 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: [...goodsFromServer],
+    currentTarget: [],
+  }
+
+toggleHandler = (good) => {
+  const copy = [...this.state.currentTarget];
+
+  if (!copy.includes(good)) {
+    this.setState({ currentTarget: [...copy, good] });
+  }
+}
+
+removeItem = (good) => {
+  this.setState((prevState) => {
+    if (prevState.currentTarget.includes(good)) {
+      prevState.currentTarget
+        .splice(prevState.currentTarget
+          .indexOf(good), 1);
+    }
+
+    return {
+      prevState: prevState.currentTarget,
+    };
+  });
+}
+
+clean = () => {
+  this.setState({ currentTarget: [] });
+}
+
+render() {
+  const { goods, currentTarget } = this.state;
+
+  return (
+    <div className="App">
+      <h1>
+        Selected good: -
+        {currentTarget.join(', ')}
+      </h1>
+      <button
+        type="button"
+        onClick={() => this.clean()}
```

Comment:
```suggestion
        onClick={this.clean}
```
A little bit shorter